lib/id3c/cli/redcap.py

```diff
@@ -196,6 +197,18 @@ def records(self, *,
         The optional *raw* parameter controls if numeric/coded values are
         returned for multiple choice fields.  When false (the default), string
         labels are returned.
+
+        The optional *page_size* parameter, when set, enables paged fetching of
```

Comment:
I think you're saying that in order to use the `page_size` parameter, the REDCap project must use for its record_id a value that is numeric and incrementing. A suggestion to make that clearer:

The optional *page_size* parameter, when set, enables paged fetching of records by their primary record id (i.e. :attr:`.record_id_field`). This parameter can be used only with REDCap projects that use a record_id that is an incrementing integer.

I dropped that auto-incremented part since we don't care how that record_id integer gets generated.

Reply:
Thanks. I've adapted your wording to clarify further and repushed.

Reply:
Thanks.